ghostwriter/reporting/templates/snippets/evidence_detail_modal.html

```diff
@@ -23,4 +23,82 @@ <h5 class="modal-title">Preview for {{ evidence.friendly_name }}</h5>
       </div>
     </div>
   </div>
-</div>
+</div> {% endcomment %}
+
+<div class="modal fade" id="evidence_detail_{{ evidence.id }}" tabindex="-1" role="dialog"
+     aria-labelledby="evidence_detail_{{ evidence.id }}_label" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="evidence_detail_{{ evidence.id }}_label">
+          {{ evidence.friendly_name }}
+        </h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <div class="container">
+          {% if evidence|get_file_type == "image" %}
+            <div class="text-center mb-3">
+              <img
+                src="{% url 'reporting:evidence_download' evidence.id %}"
```

Comment:
This should point to the evidence file directly rather than the download page, since the static file hosting will be more efficient.